fixes.inc

```diff
@@ -5142,6 +5169,50 @@ static _FIXES_DetermineOS()
 	forward bool:operator>=(File:a, _:b);
 #endif
 
+/**
+ * <remarks>
+ * Declare the function to reset the car-jacked player id to invalid id.
+ * </remarks>
+ */
+ 
+#if FIX_SilentTeleport
+	forward FIXES_RemoveJackedId(playerid);
+	public FIXES_RemoveJackedId(playerid) return FIXES_gsJackedId[playerid] = INVALID_PLAYER_ID;
+#endif
+
+/**
+ * <fixes>SilentTeleport</fixes>
+ */
+#if defined _ALS_SetVehiclePos
+     #error _ALS_SetVehiclePos defined 
+#endif 
+native BAD_SetVehiclePos(vehicleid, Float:x, Float:y, Float:z) = SetVehiclePos; 
+
+/**
+ * <fixes>SilentTeleport</fixes>
+ */
+
+#if FIX_SilentTeleport
+	stock FIXES_SetVehiclePos(vehicleid, Float:x, Float:y, Float:z)
+    {	 
+		_FIXES_FOREACH(FIXES_gsJackedId, i)
```

Comment:
It's wrong, `_FIXES_FOREACH` works differently, just use `_FIXES_FOREACH(FIXES_gsPlayersIterator, i)` for this.

Reply:
Actually if you don't mind explaining exactly how FIXES_FOREACH works that'd be great, i also think it should be explained in the style section of the documentation, many things are left out. Changed it.